config.ru

```diff
@@ -4,4 +4,9 @@ Bundler.require
 
 require "./lib/course"
 
+use Course::Middleware::LoggerMiddleware
+use Course::Middleware::ExceptionsMiddleware
+use Course::Middleware::AssetsMiddleware
+use Course::Middleware::UnsuccessMiddleware
```

Comment:
It is good to namespace in plural `middlewares`.

Reply:
If you take a look at Rack middlewares, they don't have Middleware in their name. It's just a pattern, no need to add it here too. And namespace already explicitly identifies them as middlewares.

Reply:
a habit, 'couse all SomeServics have postfix 'Servce', SomeRepository -> 'Repository'